core/email_api.php

```diff
@@ -140,6 +147,7 @@ function email_is_valid( $p_email ) {
 					return true; # no need to check mx record details (below) if we've explicity allowed the domain
 				}
 			}
+			log_event( LOG_EMAIL, "failed - not in limited domains list '$t_limit_email_domains'" );
```

Comment:
When we error out for a domain, shouldn't the message include the address that was considered invalid?